src/explore-education-statistics-frontend/next-sitemap.config.js

```diff
@@ -5,7 +5,17 @@ module.exports = {
   exclude: ['/server-sitemap.xml'],
   generateRobotsTxt: true,
   robotsTxtOptions: {
-    policies: getRobotsRuleset(process.env.APP_ENV),
+    policies: [
+      {
+        userAgent: '*',
+        allow: '/',
+        disallow: [
+          '/data-tables/fast-track/',
+          '/data-tables/permalink/',
+          '/subscriptions/',
+        ],
+      },
+    ],
```

Comment:
Following our discussion yesterday, had a little think and this will probably be fine as bots can't really crawl the non-prod environments because our basic auth will prevent general access.

Does make me think that the `meta` tags we added were probably overkill as well :thinking:  guess they don't hurt to have anyway!

Reply:
Yeah exactly - I had the same thoughts when we put them in tbh but I guess it's a good regression in case we ever remove that auth 😄